py/picca/io.py

```diff
@@ -232,16 +238,49 @@ def read_blinding(in_dir):
                                                            + '/*.fits.gz')
     filename = files[0]
     hdul = fitsio.FITS(filename)
-    if "LAMBDA" in hdul: # This is for ImageHDU format
+    # This is for ImageHDU format
+    if "LAMBDA" in hdul:
         header = hdul["METADATA"].read_header()
+
+        # read blinding
         blinding = header["BLINDING"]
-    else: # This is for BinTable format
+
+        # check if we are doing metal forests
+        if "MIN_RF_WAVE" in header and "MAX_RF_WAVE" in header:
+            lya_wave = ABSORBER_IGM["LYA"]
+            min_rf_wave = header["MIN_RF_WAVE"]
+            max_rf_wave = header["MAX_RF_WAVE"]
+            if min_rf_wave > lya_wave and max_rf_wave > lya_wave:
+                if tracer2 is None:
+                    tracer2 = lambda_abs
```

Comment:
@iprafols - where have you defined lambda_abs? Shouldn't you use here tracer2 = tracer1?

Reply:
Have you tested this code? It looks like it should have crashed...